service/service.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import dotenv from 'dotenv'
+
+import serviceRoutes from './routes/serviceRoutes.js';
+import SimpleLogger from '../utils/simpleLogger.js';
+
+const logger = new SimpleLogger();
+dotenv.config();
```

Comment:
This should be the first thing after the imports. Moreover, given that the ESM imports are resolved statically, if you want to make sure that any module can access the env variables, you should move the dotenv import and this call to a separate module/file and make it the first import.